preset/azurite/preset_test.go

```diff
@@ -0,0 +1,144 @@
+package azurite_test
+
+import (
+	"context"
+	"fmt"
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob"
+	"github.com/orlangure/gnomock/preset/azurite"
+	"github.com/stretchr/testify/assert"
+	"testing"
+
+	"github.com/orlangure/gnomock"
+)
+
+func TestPreset_Blobstorage(t *testing.T) {
+	t.Parallel()
+
+	for _, version := range []string{"latest"} {
+		t.Run(version, testBlobStorage(version))
+	}
+}
+
+func testBlobStorage(version string) func(*testing.T) {
+	return func(t *testing.T) {
+		p := azurite.Preset(
+			azurite.WithVersion(version),
+		)
+		c, err := gnomock.Start(p)
+
+		defer func() { assert.NoError(t, gnomock.Stop(c)) }()
```

Comment:
Other tests use `require` package of `testify`, would you mind replacing all `assert` calls with `require`? Or do you have a specific preference of `assert` vs `require`?

Reply:
Done in https://github.com/orlangure/gnomock/pull/837/commits/36cec276b16249cde5bb863aabfe4bc6f6170f32